leader-election-impl/src/test/java/com/palantir/paxos/PaxosStateLogMigratorTest.java

```diff
@@ -66,75 +81,104 @@ public void emptyLogMigrationSuccessfullyMarksAsMigrated() {
     }
 
     @Test
-    public void logMigrationSuccessfullyMigratesEntries() {
+    public void logMigrationWithNoLowerBoundMigratesOnlyGreatest() {
```

Comment:
note: This test is expected to change because we plan on making the migrator remember a buffer of entries before the one externally specified.